### ISSUE TITLE
version in hs.h fell out of sync again

### ISSUE THREAD

**Issue by fatalbanana** [VectorCamp/vectorscan#175]
Hi,

The patch version in hs.h is again out of sync with the release version.

Might we keep it in sync somehow please?

**Comment by markos** [comment]
You are correct, there is no reason for this file to hold this info as well as hs_version.h -which is autogenerated. This will be fixed next version.
